e2e/vpc/constants.go

```diff
@@ -11,13 +11,14 @@
 package vpc
 
 const (
-	vpcZone           = "VPC_ZONE"
-	resourceGroupID   = "RESOURCEGROUP"
-	volumeSize        = 10
-	iops              = "0"
-	volumeName        = "e2e-storage-volume"
-	volumeType        = "vpc-block"
-	generation        = "gt"
-	vpcProfile        = "general-purpose"
-	vpcConfigFilePath = "/src/github.com/IBM/ibmcloud-storage-volume-lib/e2e/config/vpc-config.toml"
+	vpcZone                = "VPC_ZONE"
+	resourceGroupID        = "RESOURCEGROUP"
+	volumeSize             = 10
+	iops                   = "0"
+	volumeName             = "e2e-storage-volume"
+	volumeType             = "vpc-block"
+	generation             = "gt"
+	vpcProfile             = "general-purpose"
+	vpcConfigFilePath      = "/src/github.com/IBM/ibmcloud-storage-volume-lib/e2e/config/vpc-config.toml"
+	volumeEncryptionKeyCRN = "ENCRYPTION_KEY_CRN"
```

Comment:
This needs to be exported before executing e2e ? same for vpcZone and resourceGroupID

Reply:
We are getting it from secret. Details are in .travis.yaml